INSTALL

```diff
@@ -587,6 +587,24 @@
 
                        ./config -DFOO CPPFLAGS=-DBAR -DCOOKIE
 
+                   Backward compatibility note:
+
+                   To be compatible with older configuration scripts, the
+                   environment variables are ignored compiling / linking flags
```

Comment:
I suppose it should read "are ignored *if* ..."

Reply:
Yup.  Fixed.